### PR TITLE
Fix Bug 1001967 - Fix RTL layout

### PR DESCRIPTION
**Pull request by kyoshino**
The site has already been localized to [Hebrew](https://webwewant.allizom.org/he/). This PR should fix the most of the RTL issues. The bar charts in the stats panel still have trouble, but better than nothing.

### CODE REVIEW

**Comment by bensternthal** [comment]
This is looking really good. I'd like @sgarrity to checkout these changes before we move forward.

File: mrburns/main/static/css/header.less [hunk 1]
```diff
@@ -156,11 +184,21 @@ header {
             width: 160px;
         }
         h2 {
-            margin-left: 200px;
+            margin: 0 0 0 200px;
```
Comment (sgarrity) [inline mrburns/main/static/css/header.less]: This is turning off top/bottom margins that should not be removed.

File: mrburns/main/static/css/stats-panel.less [hunk 2]
```diff
@@ -285,8 +286,25 @@ select.country {
...
+    .donut-icon {
+        right: 44px;
+        left: auto;
+    }
+}
+
 
 @media (min-width: @screen-sm-min) {
+    .stats-panel-contents {
+        padding: 40px 40px 40px 170px;
```
Comment (sgarrity) [inline mrburns/main/static/css/stats-panel.less]: This 40px right padding is over-ridding the 270px right padding from line #24 in this file that is needed to center the main column and keep it from overlapping the right column at some screen sizes.

**Comment by kyoshino** [comment]
Addressed @sgarrity's comments.

**Pull request merged**